packages/docs/community/contributing.md

```diff
@@ -42,9 +42,15 @@ If you are submitting an issue, please double check that there doesn’t already
 - Is tagged with the relevant labels.
 - Is assigned if you or someone else is already working on it.
 
+#### Contributing to the documentation
+
+To edit an existing page in the documentation, create a new branch for your work, commit your edits and submit a PR. 
```

Comment:
nit: External contributors won't be able to create a branch on monorepo, so they'll need to fork the repo first